packages/device_info/device_info/example/test_driver/device_info_e2e.dart

```diff
@@ -7,9 +7,6 @@ import 'package:flutter_test/flutter_test.dart';
 import 'package:device_info/device_info.dart';
 import 'package:e2e/e2e.dart';
 
-import 'package:device_info_platform_interface/device_info_platform_interface.dart';
```

Comment:
My bad, you still need the `device_info_platform_interface.dart` import:

`import 'package:device_info_platform_interface/device_info_platform_interface.dart';`

(But that's the *only* import you should need from the device_info_platform_interface package here.)